go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/SUSE/sap_host_exporter
 
-go 1.12
+go 1.11
```

Comment:
why this?

Reply:
Because we don't actually require Go 1.12, and we need to support 1.11 in SLE12-SP3.
It was initially set to 1.12 by my IDE because I have that version installed in my workstation. ;)

Reply:
ok